### DISCUSSION TITLE
localsend in SEND TO (win 10)

### DISCUSSION THREAD

**Discussion by flyjodel** [localsend/localsend#450] [Discussion (English, International)]
Hi, 
very nice app !
Is it possible, and how, to add LocalSend to the SendTo folder of Win10, so I can send a file with a rightclic ?
If i add the localsend.app shortcut to this folder, there are some authorization issues that prevent somebody to send files with a rightclic
Thanks

**Comment by BBlackBunny** [comment]
i really hope they add this function too! or add to "share" menu

**Reply by assiless** [reply]
i prefer share https://github.com/localsend/localsend/issues/1065#issuecomment-1878544252

**Comment by xlandhenry** [comment]
Share dialog is the way to go for Windows 11 and above.

**Reply by paolosezart** [reply]
However, it works fine in Daemon tools "Catch!" program. However, it is not available in linux (I found only windows and android). Besides, it is a paid program. Nevertheless, this functionality is possible in windows 7 and younger versions of windows.

**Comment by Koboku** [comment]
Please add this option, either directly as a new option in context menu or through Send to.

I think that sending a file should take minimal amount of actions. So it's why we have option to choose save folder and favorite devices list, but it can be even quicker:

- No need to open a mobile app, it runs in the background
- Send from PC context menu
- Send Directly to one default device, without choosing it from the list (additional context menu option)

**Comment by chenshaoju** [comment]
If you are using the portable version, all you have to do is drag and drop the shortcut of localsend_app.exe into the ```%USERPROFILE%\AppData\Roaming\Microsoft\Windows\SendTo``` directory, like this:
![image](https://github.com/user-attachments/assets/aed50310-e2c0-4271-ae21-670f4ab03ba5)

After the operation is completed, you can use it in the Send To menu:
![image](https://github.com/user-attachments/assets/834217c9-5fb6-4b0b-a682-3f4bafda355f)

![image](https://github.com/user-attachments/assets/1819f5fc-9d80-46d7-8a29-6be679241f13)
